cpp/src/cider/type/data/CiderTypes.h

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
rename this file?

Reply:
Actually this content should be consolidated in other file and this file should be removed. This can be done in late refactor.